Frontend/src/components/Generate.jsx

```
@@ -18,46 +16,53 @@ const Generate = () => {
 
   useEffect(() => {
     fetchLatestImage();
-  }, []);
+  }, [fetchLatestImage]); // ✅ Correct usage
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)